Assets/Scripts/InventoryScripts/ManagerScript.cs

```diff
@@ -5,80 +5,59 @@
 
 public class ManagerScript : MonoBehaviour {
 
-    public GameObject invPanel, journalInvPanel, itemInfo;    //reference to a GameObject
-    public InventoryScript inventoryScript;                     //reference to InventoryScript
-    public JournalInventoryScript journalScript;
-    [SerializeField] Text nameOfItem, descriptionOfItem;        //Serialize Text fields for information panel
-    [HideInInspector]
-    public bool active = false;    // bool to check if inventory is open or closed
+    [SerializeField] private GameObject _invPanel, _journalInvPanel, _itemInfo;    //reference to a GameObject
+    [SerializeField] private InventoryScript _inventoryScript;                     //reference to InventoryScript
+    [SerializeField] private JournalInventoryScript _journalScript;
+    [SerializeField] private Text _nameOfItem, _descriptionOfItem;        //Serialize Text fields for information panel
+    public bool Active { get; set; }    // bool to check if inventory is open or closed
```

Comment:
Shouldn't this be a private setting property? Doesn't seem like something other classes should be allowed to change on a whim.